docs/moduleAna5Utils.rst

```diff
@@ -0,0 +1,105 @@
+
+#####################################################################
+ana5Utils
+#####################################################################
+
+
+distanceTimeAnalysis: Visualizing the temporal evolution of flow variables
+------------------------------------------------------------------------------
+
+With the functions gathered in this module, flow variables of avalanche simulation results can be
+visualized in a distance versus time diagram, the so called **thalweg-time diagram**.
+The **tt-diagram** provides a way to identify main features of the temporal evolution of
+flow variables along the :term:`avalanche path`.
+This is based on the ideas presented in :cite:`FiFrGaSo2013` and :cite:`RaKo2020`, where
+avalanche simulation results have been transformed into the radar coordinate system to facilitate
+direct comparison, combined with the attempt to analyze simulation results in an avalanche path
+dependent coordinate system (:cite:`Fi2013`).
+In addition to the **tt-diagram**, :py:mod:`ana5RangeTime` also offers the possibility to
+produce simulated **range-time diagrams** of the flow variables with respect to a radar field
+of view. With this, simulation results can be directly compared to radar measurements in terms
+of front position and inferred approach velocity. The colorcoding of the simulated
+**range-time** diagrams refers to average values of the chosen flow parameter
+(e.g. flow depth, flow velocity) at specified range gates. This colorcoding is not directly
+comparable to the intensity given in the range-time diagram from radar measurements.
+
+.. Note::
+  The data processing for the **tt-diagram** and the **range-time diagram** can be done
+  *during run time* of :py:mod:`com1DFA`, or as a postprocessing step. However, the second option
+  requires saving all the required time steps of the flow variable fields, which is much more
+  computationally expensive compared to the first option.
+
+To run
+~~~~~~~
+
+During run-time of :py:mod:`com1DFA`:
+
+* in your local copy of ``com1DFA/com1DFACfg.ini`` in [VISUALISATION] set `createRangeTimeDiagram`
+  to True and chose if you want a *TTdiagram* by setting this flag to True or in the case of a
+  simulated range-time diagram to False
+
+* in your local copy of ``ana5Utils/distanceTimeAnalysisCfg.ini`` you can adjust the default settings
+  for the generation of the diagrams
+
+* run  :py:mod:`runScripts.runThalwegTimeDiagram.py` or :py:mod:`runScripts.runRangeTimeDiagram.py`
+  and set the `preProcessedData` flag to `True`
+
+As a postprocessing step:
+
+* have a look at :py:mod:`runScripts.runThalwegTimeDiagram.py` and :py:mod:`runScripts.runRangeTimeDiagram.py`
+
+* in your local copy of ``ana5Utils/distanceTimeAnalysisCfg.ini`` you can adjust the default settings
+  for the generation of the diagrams
+
+The resulting figures can be found in ``avalancheDirectory/Outputs/ana5Utils``.
+
+
+.. figure:: _static/thalwegTime_FD.png
```

Comment:
Approach velocity and value of max velocity have opaque white and not completely transparent background...

Reply:
yep this is on purpose to make it 'more' visible but not fully hide the underlying lines..